src/main/kotlin/kweb/Kweb.kt

```diff
@@ -169,45 +169,39 @@ class Kweb private constructor(
     There is a bit of duplication here, because we have to pass in the javascript string separately.
     We need it to create the debugToken. Some server2ClientMessages will contain the javascript,
      but messages that call cached functions will not have it. So we have to make sure we pass it in separately.*/
-    fun callJs(server2ClientMessage: Server2ClientMessage, javascript: String) {
-        val wsClientData = clientState.getIfPresent(server2ClientMessage.yourId)
-                ?: error("Client id ${server2ClientMessage.yourId} not found")
+    fun callJs(sessionId: String, funcCall: FunctionCall, javascript: String) {
+        val wsClientData = clientState.getIfPresent(sessionId)
+                ?: error("Client id $sessionId not found")
         wsClientData.lastModified = Instant.now()
         val debugToken: String? = if(!debug) null else {
             val dt = abs(random.nextLong()).toString(16)
             wsClientData.debugTokens[dt] = DebugInfo(javascript, "executing", Throwable())
             dt
         }
-        server2ClientMessage.debugToken = debugToken
+        funcCall.debugToken = debugToken
```

Comment:
Not necessary to fix, but might be good to add a TODO to figure out how to make debugToken immutable.